### PR TITLE
📖 Specifies adding env variable for java version

### PR DESCRIPTION
**Pull request by Enriqe**
Related to #15119.

Adds specifying java version in env variables of bash profile.

### CODE REVIEW

File: contributing/getting-started-e2e.md [hunk 1]
```diff
@@ -173,6 +173,7 @@ Now that you have all of the files copied locally you can actually build the cod
   An alternative to installing `yarn` is to invoke each Yarn command in this guide with `npx yarn` during local development. This will automatically use the current stable version of `yarn`.
 
 * Closure Compiler is automatically installed by Yarn, but it requires Java 8 which you need to install separately. AMP's version of Closure Compiler won't run with newer versions of Java. Download an installer for Mac, Linux or Windows [here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+  * Note: If you have several versions of Java installed, make sure to be using Java 8 by updating the environment variable in your bash profile: ```export JAVA_HOME=`/usr/libexec/java_home -v 1.8```.
```
Comment (rsimha) [inline contributing/getting-started-e2e.md]: The triple backticks should be in separate lines for the code be shown as a callout like this:
```
export JAVA_HOME=`/usr/libexec/java_home -v 1.8
```
So you'll need to write this as follows (minus the spaces):

```
` ` `
export JAVA_HOME=`/usr/libexec/java_home -v 1.8
` ` `

```

Comment (rsimha) [reply contributing/getting-started-e2e.md]: Also restate the text to:

```
If you have multiple versions of Java installed, make sure you are using Java 8 by adding this to `~/.bashrc`:
````

File: contributing/getting-started-e2e.md [hunk 2]
```diff
@@ -173,6 +173,11 @@ Now that you have all of the files copied locally you can actually build the cod
   An alternative to installing `yarn` is to invoke each Yarn command in this guide with `npx yarn` during local development. This will automatically use the current stable version of `yarn`.
 
 * Closure Compiler is automatically installed by Yarn, but it requires Java 8 which you need to install separately. AMP's version of Closure Compiler won't run with newer versions of Java. Download an installer for Mac, Linux or Windows [here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+  * Note: If you have multiple versions of Java installed, make sure you are using Java 8 by adding this to `~/.bashrc`:
+
+  ```
+  export JAVA_HOME=`/usr/libexec/java_home -v 1.8
```
Comment (rsimha) [inline contributing/getting-started-e2e.md]: I see an unpaired backtick before the path. 

Also, does this work on all platforms? If not, you need to specify that it's mac OS only.

Comment (Enriqe) [reply contributing/getting-started-e2e.md]: Was only able to test with mac, so I added that as part of the note.

**Comment by Enriqe** [comment]
@rsimha can you merge this since I don't have the permissions? Thanks!

**Comment by Enriqe** [comment]
friendly ping @rsimha

**Pull request merged**